frontend/src/pages/accounts/[id].jsx

```diff
@@ -32,7 +34,7 @@ export default class extends React.Component {
 
   render() {
     const { account, accountFetchingError, currentNearNetwork } = this.props;
-
+    Mixpanel.track("View Individual Account", { accountId: account.accountId });
```

Comment:
Should we move it to `componentDidMount`?

Reply:
No, I test it and it works fine?

Reply:
Why do we need to call it on every render event?